samples/build/build_source-to-image_cr.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   name: s2i-nodejs-build
 spec:
   source:
-    url: https://github.com/sclorg/nodejs-ex
+    url: https://github.com/shipwright-io/sample-nodejs
+    contextDir: source-build
```

Comment:
The source-to-image build strategy to my knowledge does not support `contextDir`. Is maybe easy to extend, but for the moment, maybe retain the old repo and address this separately?

Reply:
`s2i build --as-dockerfile` also has support for `--context-dir`, I can try adding a PR to update.

Reply:
@adambkaplan I think you can PR that, in the meantime I will not add the new repo here.

Reply:
Resolving in favor of https://github.com/shipwright-io/build/issues/638